### PR TITLE
feat(LIB-392): init topbar component

### PR DESCRIPTION
**Pull request by oxcened**
Jira: [LIB-392](https://fiscozen.atlassian.net/browse/LIB-392)

### CODE REVIEW

**Review by Trebh — APPROVED** [review]
lgtm

File: packages/topbar/src/FzTopbar.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<template>
+  <div :class="containerClass">
+    <span> Testo qui </span>
```
Comment (Crish15) [inline packages/topbar/src/FzTopbar.vue:3]: You should probably use `<slot></slot>` here and not this static text

**Review by Crish15 — CHANGES_REQUESTED** [review]

File: packages/topbar/src/FzTopbar.vue (hunk 1 above)
Comment (oxcened) [reply packages/topbar/src/FzTopbar.vue:3]: oh yeah my bad, good catch

Comment (oxcened) [reply packages/topbar/src/FzTopbar.vue]: done in 13063ef

**Review by Crish15 — APPROVED** [review]

**Pull request merged**